superdesk/commands/index_from_mongo.py

```diff
@@ -68,10 +69,11 @@ def copy_resource(cls, resource, page_size, last_id):
                 else:
                     break
 
-            if last_id:
-                print(last_id)
-
             print("{} Inserted {} items in {:.3f} seconds".format(time.strftime("%X %x %Z"), success, time.time() - s))
+
+            # print last ID for items
+            print(items[-1][config.ID_FIELD])
```

Comment:
I think this actually prints after each bucket but you only want to do it at the very end when this `for items ...` cycle is done